posthog-node/src/client.ts

```diff
@@ -639,6 +648,43 @@ export abstract class PostHogBackendClient extends PostHogCoreStateless implemen
     return super._shutdown(shutdownTimeoutMs)
   }
 
+  private async _requestRemoteConfigPayload(flagKey: string): Promise<PostHogFetchResponse | undefined> {
+    if (!this.options.personalApiKey) {
+      return undefined
+    }
+
+    const url = `${this.host}/api/projects/@current/feature_flags/${flagKey}/remote_config/`
+
+    const options: PostHogFetchOptions = {
+      method: 'GET',
+      headers: {
+        ...this.getCustomHeaders(),
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${this.options.personalApiKey}`,
+      },
+    }
```

Comment:
Why not use `getPersonalApiKeyRequestOptions` here like before?

Reply:
ehhh, it was localize the the `FeatureFlagsPoller` class and I couldn't be bothered to factor it out.